libs/execution_context/unit/execution_context_unit_test.cpp

```diff
@@ -16,6 +16,7 @@
 #include <optional>
 #include <set>
 
+#if !defined(CONFIG_BPF_JIT_DISABLED)
 typedef struct _free_trampoline_table
```

Comment:
can CONFIG_BPF_JIT_DISABLED code be moved to a separate file instead of ifdef'ed in the same file?